namastack-outbox-core/src/main/kotlin/io/namastack/outbox/handler/OutboxHandlerBeanPostProcessor.kt

```diff
@@ -82,7 +88,10 @@ internal class OutboxHandlerBeanPostProcessor(
     ): Any {
         scanners
             .flatMap { it.scan(bean) }
-            .forEach { handlerMethod -> handlerMethod.register(registry) }
+            .forEach { handlerMethod ->
```

Comment:
Consider renaming the `handlerMethod.register` method to something like `handlerMethod.registerHandler` or `handlerMethod.selfRegister`.